### DISCUSSION TITLE
Feature Flag?

### DISCUSSION THREAD

**Discussion by xiangninglyu** [mlflow/mlflow#12574] [Ideas]
Hey folks, have we started support for the feature flagging functionality? I work for a company that uses MLflow but still Statsig for experimentation because the latter support feature flags. Would really love to have MLflow solely.

**Comment by B-Step62** [comment]
@xiangninglyu Would you mind elaborating what the feature flags capability is in Statsig? How will it relate to MLflow use cases?
